.github/workflows/reviewdog.yml

```diff
@@ -33,3 +38,40 @@ jobs:
           flake8 | \
             reviewdog -f=pep8 -name=flake8 \
               -tee -reporter=github-check -filter-mode nofilter
+
+  mypy:
+    name: "Mypy"
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+
+      - name: Set up reviewdog
+        run: |
+          mkdir -p "$HOME/bin"
+          curl -sfL \
+            https://github.com/reviewdog/reviewdog/raw/master/install.sh | \
+              sh -s -- -b "$HOME/bin"
+          echo "$HOME/bin" >> $GITHUB_PATH
+
+      - name: Install Python dependencies
+        run: |
+          python -m pip install --upgrade pip setuptools wheel
+          python -m pip install --upgrade mypy
```

Comment:
Do we want to pin mypy here?

In the main repo we pin because they broke CI, and certainly possible that they could here as well, though less problematic for red CI here, as the velocity of change is a lot lower...